src/main/java/Duke.java

```diff
@@ -1,6 +1,9 @@
 import duke.command.Command;
 import duke.exception.DukeException;
-import duke.util.*;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
 
 public class Duke {
```

Comment:
Missing a header comment for Duke class? (from what I understand every public class/ method needs to have a javadoc comment).